config/initializers/geocoder.rb

```diff
@@ -1,11 +1,10 @@
 redis = Redis.new(url: ENV['REDIS_CACHE_URL'])
-key = ENV.fetch('GOOGLE_GEOCODING_API_KEY', '')
+key = ENV.fetch('OS_GEOCODING_API_KEY', '')
 
 Geocoder.configure(
-  lookup: :google,
+  lookup: :os_names,
   api_key: key,
   timeout: 5,
-  units: :mi,
```

Comment:
Does OS default to miles or something?

Reply:
Whoops - dunno why I deleted this. It's not actually relevant to the Geocoding part of the gem, it's used for calculations (https://github.com/alexreisner/geocoder#geospatial-calculations), which we don't use, but may use in future.